romeo/src/state.rs

```diff
@@ -105,7 +131,7 @@ where
 pub fn bootstrap(state: &State) -> Task {
     match state.contract {
         None => Task::CreateAssetContract,
-        Some(_) => Task::FetchBitcoinBlock(state.block_height),
+        Some(_) => Task::FetchBitcoinBlock(state.block_height.map(|block_height| block_height + 1)),
```

Comment:
Only because you do this in two different spots (line 134 and line 174), it might be worthwhile making this a helper function like "next_block_height" to prevent a typo down the line.

Reply:
Yep can do that 👍